docs/specs/build/toc/basics.yml

```diff
@@ -309,3 +310,7 @@ outputs:
        ]
     }
   build.manifest:
+  build.log: |
+    ["warning","referenced-content-outof-scope","the referenced content file docs/f1/f11/n1.md is outof config's scope"]
```

Comment:
nit: surround file path with '' in error/warning message in case there are spaces in file names